portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -681,6 +681,18 @@ definition {
 		AssertElementPresent(locator1 = "Form#ALT_PAGINATION_SUCCESS_PAGE");
 	}
 
+	macro viewFieldReference {
+		key_fieldLabel = "${fieldLabel}";
+
+		FormViewBuilder.clickFieldLabel(fieldLabel = "${key_fieldLabel}");
+
```

Comment:
Hi @evanilson-santana , I just noticed that FormViewBuilder.clickFieldLabel macro belongs to AppBuilder, not to Forms. Maybe you could try the Click function. Please, check if this example I found in some testcases is applicable to this case.

```
Click(key_name = "Numeric", locator1 = "FormFields#FIELD_TARGET_LABEL");
```
Thanks! =)